src/Microsoft.Azure.Batch.SoftwareEntitlement.Server/Controllers/SoftwareEntitlementsController.cs

```diff
@@ -85,6 +111,19 @@ public SoftwareEntitlementsController(Options options, ILogger logger)
             return Ok(response);
         }
 
+        /// <summary>
+        /// Check to see whether the specified <c>api-version</c> is valid for software entitlements
+        /// </summary>
+        /// <param name="apiVersion">Api version from the query parameter</param>
+        /// <returns>True if it is valid, false otherwise.</returns>
+        private bool IsValidApiVersion(string apiVersion)
+        {
+            // Check all the valid apiVersions
+            // TODO: Once this list passes three or four items, use a HashSet<string> to do the check more efficiently
+            return apiVersion.Equals("2017-05-01.5.0", StringComparison.Ordinal)
```

Comment:
[Minor ✍️] This is working a bit harder than it needs to, plain `==` does ordinal comparison.